pkg/controller/test/context_builder.go

```diff
@@ -294,7 +294,7 @@ func (b *Builder) AllActionsExecuted() error {
 }
 
 func actionToString(a coretesting.Action) string {
-	return fmt.Sprintf("%s %q in namespace %s", a.GetVerb(), a.GetResource(), a.GetNamespace())
+	return fmt.Sprintf("%s %s %q in namespace %s", a.GetVerb(), a.GetSubresource(), a.GetResource(), a.GetNamespace())
```

Comment:
This makes it easier to see whether the missing action is an Update or an UpdateStatus operation.